features/shared-storage-non-cookie.yml

```diff
@@ -1,7 +1,7 @@
-draft_date: 2024-09-09
 name: Extending Storage Access API (SAA) to non-cookie storage
-description: TODO
+description: "This extension of the Storage Access API enables content in cross-site iframes to request access to first-party data beyond cookies."
 spec: https://privacycg.github.io/saa-non-cookie-storage/
+group: storage
 compat_features:
   - api.Document.hasUnpartitionedCookieAccess
```

Comment:
Move this key to `storage-access`. It's an alias to `document.hasStorageAccess()`.